reference/sql_workarounds.rst

```diff
@@ -124,41 +124,74 @@ There is a workaround: you can replicate the local table to a single shard on ev
 This will create a table with a single shard (non-distributed), but will
 replicate that shard to every node in the cluster. Now Citus will accept a join query between *here* and *there*, and each worker will have all the information it needs to work efficiently.
 
+.. _window_func_workaround:
+
+Window Functions
+----------------
+
+Currently Citus does not have out-of-the-box support for window functions on queries involving distributed tables, but there is a straightforward workaround. Window functions will work on a distributed table if
```

Comment:
So this is true only for cross-shard queries. For multi-tenant cases where we hit only a single tenant, this would work. Could you update this comment to reflect this in a stronger manner?

Reply:
I realize now we have a comment up top in the section saying we support full SQL for multi-tenant workloads, but this statement still is quite misleading: "Currently Citus does not have out-of-the-box support for window functions". 
We could also rename this as "Cross-shard queries with Window Functions" or something similar to drive home the point.